app/forms/declaration/CommodityMeasure.scala

```diff
@@ -29,12 +29,12 @@ object CommodityMeasure {
 
   val mapping = Forms.mapping(
     "supplementaryUnits" -> optional(text().verifying("supplementary.commodityMeasure.supplementaryUnits.error", validateDecimal(16)(2))),
-    "netMass" -> text
-      .verifying("supplementary.commodityMeasure.netMass.empty", nonEmpty)
-      .verifying("supplementary.commodityMeasure.netMass.error", isEmpty or validateDecimal(11)(3)),
     "grossMass" -> text
```

Comment:
Why are we changing the order of the validation?

Reply:
To maintain uniformity